Sources/Prometheus/MetricTypes/Gauge.swift

```diff
@@ -84,29 +86,31 @@ public class Gauge<NumType: Numeric, Labels: MetricLabels>: Metric, PrometheusHa
     /// - Parameters:
     ///     - amount: Amount to increment the Gauge with
     ///     - labels: Labels to attach to the value
+    ///     - done: Completion handler
     ///
     public func inc(_ amount: NumType, _ labels: Labels? = nil, _ done: @escaping (NumType) -> Void = { _ in }) {
         prometheusQueue.async(flags: .barrier) {
-        if let labels = labels {
-            var val = self.metrics[labels] ?? self.initialValue
-            val += amount
-            self.metrics[labels] = val
-            done(val)
-        } else {
-            self.value += amount
-            done(self.value)
-        }
+            if let labels = labels {
+                var val = self.metrics[labels] ?? self.initialValue
+                val += amount
+                self.metrics[labels] = val
+                done(val)
+            } else {
+                self.value += amount
+                done(self.value)
+            }
         }
     }
     
     /// Increments the Gauge
     ///
     /// - Parameters:
     ///     - labels: Labels to attach to the value
+    ///     - done: Completion handler
     ///
     public func inc(_ labels: Labels? = nil, _ done: @escaping (NumType) -> Void = { _ in }) {
-        self.inc(1, labels) {
-            done($0)
+        self.inc(1, labels) { value in
+            done(value)
```

Comment:
would you mind changing this to
```swift
self.dec(1, labels, done)
```
instead? Not sure why I did not do that in the first place